### PR TITLE
Save active tab in settings upon every tab switch

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

**Comment by tworaz** [comment]
LGTM

File: src/persistenttabmodel.cpp [hunk 1]
```diff
@@ -70,6 +68,8 @@ void PersistentTabModel::tabsAvailable(QList<Tab> tabs)
         m_loaded = true;
         emit loadedChanged();
     }
+
+    connect(this, SIGNAL(activeTabIndexChanged()), this, SLOT(saveActiveTab()));
```
Comment (rainemak) [inline src/persistenttabmodel.cpp]: This should be unique connection as tabsAvailable will be emitted also when all tabs are closed.

**Comment by rainemak** [comment]
Commit sha1 a829460 has a broken change log entry and the activeTabIndexChanged connection should be unique. Please fix those before merging. Other than that LGTM

**Comment by rojkov** [comment]
Updated the activeTabIndexChanged connection to be unique and shortened the commit message.

**Comment by rainemak** [comment]
LGTM

**Pull request merged**